### PR TITLE
11.6.4 What's New

### PR DESCRIPTION
**Pull request by aubergine10**
Airports DLC release.

* [New] TM:PE 11.6 branch released as STABLE - huge thanks to our beta testers!
* [Mod] Fixed: `Move It` traffic settings not copy/pasted in-game #1303 #1300 (krzychu124)
* [Mod] Fixed: `Node Controller Renewal` silent fail place intersection #1308 #1307 (krzychu124)
* [Mod] Fixed: `Intersection Marking Tool` silent fail place intersection #1308 #1307 (krzychu124)
* [Updated] Compatibility update for Airports DLC #1230 (krzychu124)
* [Updated] Reduce logging from `AssetDataExtension` in release builds #1296 #1295 (aubergine18)
* [Updated] Translations for multiple languages #1304 (Chamëleon, MenschLennart, John Deehe, Skazov, AlexofCA, CrankyAnt, Иван Соколов, AduitSSH, John Lok Ho, DNSE, shg166, GiorgioHerbie, footbool, krzychu124, aubergine18, Dmytro Lytovchenko, Fizzy_LaFizz, Vladimír Krajč, alvaroer, Mahmoud Al Gammal, ipd, 田七不甜, vitalii201, 許景翔, Mehmet Ömer Tüzer, vicugna-pacos, kianzarrin, Mbyron26, Headspike, lennier3, kevinlin18, Ikki_di_Phoenix)
* [Updated] Reorganise General options tab #1306 #1262 (aubergine18)
* [Updated] Improved compatibility checks, inhibit `Resume` if problems detected #1310 #1309 #699 #697 (aubergine18)

### CODE REVIEW

**Comment by aubergine10** [comment]
@krzychu124 there's no release tag for 11.6.3

**Comment by krzychu124** [comment]
Yup... I'll push it later today and create a release.

**Comment by aubergine10** [comment]
- Always log expected vs. actual C:SL ver in TMPE.log
- Bump mod versions to `11.6.4`
- Bump game version to `1.14.0.4` / `193061904u`
- Wiki release guide updated as it was missing some info: https://github.com/CitiesSkylinesMods/TMPE/wiki/Release-Binaries

**Review by kvakvs — APPROVED** [review]
Nice

**Comment by krzychu124** [comment]
> Yup... I'll push it later today and create a release.

Done

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,16 +29,39 @@ This changelog includes all versions and major variants of the mod going all the
 
 </details>
 
+#### TM:PE V11.6.4.0 STABLE, 25/01/2022
+
+* [New] TM:PE 11.6 branch released as STABLE - huge thanks to our beta testers!
+* [Mod] Fixed: `Find It` traffic settings not copy/pasted in-game #1303 #1300 (krzychu124)
```
Comment (krzychu124) [inline CHANGELOG.md:35]: I think you meant Move it 🙃

Comment (aubergine10) [reply CHANGELOG.md:35]: yup, good catch, will update changelogs

**Comment by aubergine10** [comment]
@krzychu124 assuming there will be a PR(s?) for changes due to game update, the following files will need updating with # references:

* `whats_new.txt`
* `README.md` - both STABLE and TEST
* `CHANGELOG.md` - both STABLE and TEST

There's already an entry `* [Updated] Compatibility update for Airports DLC #1230 (krzychu124)` which the PR #('s?) can be added to.

**Comment by aubergine10** [comment]
Image for workshop page (assume you already have this, but just in case: https://i.imgur.com/wEQMnVA.gif )
img[/img]

**Comment by krzychu124** [comment]
What about this (shared by BadPeanut, has transparent background)
1 14-mod-badge

**Comment by aubergine10** [comment]
yup, even better :)

**Comment by krzychu124** [comment]
> There's already an entry * [Updated] Compatibility update for Airports DLC #1230 (krzychu124) which the PR #('s?) can be added to.

Theoretically today release day (will be released 10am, now it's 2am here) I can push changes here, but Appveyor requires an update of dlls because it won't compile due to breaking change in one of the methods

**Review by krzychu124 — APPROVED** [review]
👍 Releasing...

**Pull request merged**